tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -6,9 +6,9 @@ Feature: spaces management
       | Alice |
       | Brian |
     And "Admin" assigns following roles to the users
```

Comment:
only comment, we can do it in separate PR.
maybe change it as: `And "Admin" assigns following roles to the users using API`?
and change another steps from https://github.com/owncloud/web/blob/master/tests/e2e/cucumber/steps/api.ts